src/components/cards/bookingRequestCard/PEBookingRequestCardCustomers.tsx

```diff
@@ -44,7 +44,13 @@ export default function PEBookingRequestCardCustomers({
             <div className="flex gap-4">
                 <div className={'overflow-hidden rounded-3'}>
                     {clientImage ? (
-                        <Image src={clientImage ?? ''} alt={'client image'} width={45} height={45} />
+                        <Image
+                            style={{ width: '100%', objectPosition: 'center', objectFit: 'cover' }}
+                            src={clientImage ?? ''}
```

Comment:
Client image cannot be undefined here
```suggestion
                            src={clientImage}
```